README.md

```diff
@@ -188,36 +188,52 @@ Optional: `true`
 Configure Sugarcoat's logging properties. See [npm/npmlog](https://github.com/npm/npmlog#loglevel) for more info.
 
 ### `template.cwd` ###
- 
-Type: `String`  
-Optional: `true`  
+
+Type: `String`
+Optional: `true`
 Default: Sugarcoat's theme directory
 
-The base path to which all `template` paths are relative. 
+The base path to which all `template` paths are relative.
 
 ### `template.layout` ###
 
-Type: `String`  
-Optional: `true`  
+Type: `String`
+Optional: `true`
 Default: `main.hbs` (provided by Sugarcoat).
 
 Path (relative to `template.cwd`, if provided) to the Handlebars layout that will define the layout of the site.
 
 ### `template.partials` ###
 
-Type: [Standardized File Format](#standardized-file-format)  
+Type: Array of [Standardized File Format](#standardized-file-format)
 Optional: `true`
 Default: See [templating](#templating) for a list of Sugarcoat's provided partials.
 
-A standardized file format of one or more directory (not file) paths (relative to `template.cwd`) to register with Handlebars. If any partials use a [reserved name](#reserved-partial-names), the respective partial will override the one provided by Sugarcoat. If you choose to include an object or an array of objects, you must include a `src` and `options`. If you do not choose to include options through an object, Sugarcoat will default it's glob options to `nodir: true`.
+File(s) (relative to `template.cwd`) to register with Handlebars. If any partials use a [reserved name](#reserved-partial-names), the respective partial will override the one provided by Sugarcoat. If you choose to include an object or an array of objects, you must include a `src` and `options`. If you do not choose to include options through an object, Sugarcoat will default it's glob options to `nodir: true`.
```

Comment:
Seems a little confusing that the `type` is an array of standardized file format, but then you can include an object. Is there a better way of wording this?